src/situations/commun/vues/bouton_lecture.vue

```diff
@@ -41,6 +45,10 @@ export default {
     nomTechnique: {
       type: String,
       required: true
+    },
+    avecTexte: {
+      type: Boolean,
+      required: false
```

Comment:
Je mettrais bien une valeur par défaut à ce props :) 

```suggestion
    avecTexte: {
      type: Boolean,
      required: false,
      default: false
```